config/ompi_check_pvfs2.m4

```diff
@@ -46,70 +46,27 @@ AC_DEFUN([OMPI_CHECK_PVFS2],[
              [Build Pvfs2 support, optionally adding DIR/include, DIR/lib, and DIR/lib64 to the search path for headers and libraries])])
```

Comment:
Are the variables above this used at all any more (e.g., the `_save_` variables)?